coordinator/events/eventlog.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Edgeless Systems GmbH.
+//
```

Comment:
Please run `gofmt -w coordinator/events/eventlog.go` to format this file.

Also all top-level, exported types and functions should have doc comments.